seed/views/main.py

```diff
@@ -2376,16 +2377,18 @@ def save_gb_request_info(request):
     if time_type == 'date':
         last_date = yesterday.strftime(date_pattern)
         if loopback_flag == 'Y':
-            very_first_date = datetime.strptime('1/1/2014', '%m/%d/%Y').date()
+            very_first_date = datetime.strptime('3/1/2014', '%m/%d/%Y').date()
             last_date = very_first_date.strftime(date_pattern)
     else:
         last_date = str(calendar.timegm(time.strptime(yesterday.strftime('%m/%d/%Y'), '%m/%d/%Y')))
         if loopback_flag == 'Y':
-            last_date = str(1388534400)
+            last_date = str(1393632000)
```

Comment:
I will add the comments.

On Thu, Mar 3, 2016 at 4:05 PM, Nicholas Long <notifications@github.com>
wrote:

> In seed/views/main.py
> <https://github.com/SEED-platform/seed/pull/817#discussion_r54948152>:
>
> >              last_date = very_first_date.strftime(date_pattern)
> >      else:
> >          last_date = str(calendar.timegm(time.strptime(yesterday.strftime('%m/%d/%Y'), '%m/%d/%Y')))
> >          if loopback_flag == 'Y':
> > -            last_date = str(1388534400)
> > +            last_date = str(1393632000)
>
> can you add a comment to what this "last_date" is?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SEED-platform/seed/pull/817/files#r54948152>.
>